manifests/init.pp

```diff
@@ -307,10 +312,15 @@
   }
 
   case $::operatingsystem {
-    'Ubuntu','Debian' : { include datadog_agent::ubuntu }
+    'Ubuntu','Debian' : {
```

Comment:
As discussed, this could be a breaking change for people specifying the version on `ubuntu`. We should probably clean up, decide if we keep the old way where we just `include datadog_agent::ubuntu` versus specifying the class.